### PR TITLE
Fix: merge conflicts in README update

### PR DESCRIPTION
**Pull request by yo1995**
This PR solves the merge conflicts in pRU, by updating the conflicted files to latest version. 
The only thing I did is to use latest docs in #923 to override these conflicts.

Related to #869

### CODE REVIEW

File: arcgis-ios-sdk-samples/Features/Feature layer selection/README.md [hunk 1]
```diff
@@ -17,8 +17,8 @@ Tap on a feature in the map. All features within a given tolerance (in pixels) o
 
 1. Create an `AGSServiceFeatureTable` from a feature service URL.
 2. Create an `AGSFeatureLayer` from the service feature table.
-3. Identify nearby features at the tapped location by specifying the `AGSGeometry` for the `AGSQueryParameters`.
-4. Select all identified features in the feature layer with `AGSFeatureLayer.selectFeatures(withQuery:mode:completion:)`.
+3. Identify nearby features at the tapped location using `AGSGeoView.identifyLayer(_:screenPoint:tolerance:returnPopupsOnly:maximumResults:completion:)` method on the map view.
```
Comment (philium) [inline arcgis-ios-sdk-samples/Features/Feature layer selection/README.md:20]: It seems like this should either be:
```suggestion
3. Identify nearby features at the tapped location using `AGSGeoView.identifyLayer(_:screenPoint:tolerance:returnPopupsOnly:maximumResults:completion:)` on the map view.
```
or:
```suggestion
3. Identify nearby features at the tapped location using the `AGSGeoView.identifyLayer(_:screenPoint:tolerance:returnPopupsOnly:maximumResults:completion:)` method on the map view.
```

**Comment by yo1995** [comment]
@philium These changes look good, but might not be part of this PR.

merge conflict

I created this PR to solve the merge conflicts in #869. Since I shouldn't make changes on `philium/readme-updates` branch directly, I need to either merge from the latest `v.next` to solve the conflicts (which is not easy to review), or update only these 3 conflicting files with their `v.next` counterparts, so there are not conflicts.

I choose the latter which is easier to handle and review. So it's better to keep them as is, and we may make these changes after #869 get merged. What do you think? 

When #869 is ready to merge, I'll ping you and let you merge it as you opened that PR.

**Comment by philium** [comment]
> I choose the latter which is easier to handle and review.

That option may make review easier, but I don't know what other consequences it may have. I think there's a reason why GitHub doesn't suggest fixing merge conflicts by just manually editing the files in conflict without first updating them from the target branch. I would prefer it if the conflicts were resolved in the traditional way (and the way we've done it for the `philium/readme-updates` branch in the past) by branching from it, updating that branch from `v.next`, resolving conflicts in that branch, and then creating a PR to merge the branch back into `philium/readme-updates`.

**Comment by yo1995** [comment]
> I would prefer it if the conflicts were resolved in the traditional way (and the way we've done it for the `philium/readme-updates` branch in the past) by branching from it, updating that branch from `v.next`, resolving conflicts in that branch, and then creating a PR to merge the branch back into `philium/readme-updates`.

I did that in #931 , and there were >200 changes, so I wasn't confident about it, even if I believe I handled it correctly...

Do you think that is the way to go? I can reopen that PR if so.

Also I always wondered how should we review these "reverse merge" PRs with many commits and changes?

**Comment by philium** [comment]
> Do you think that is the way to go? I can reopen that PR if so. Or should I use rebase?

I do think that is the way to go. I wouldn't mind if you reopened your first PR, although I do have a question about the title. What in the world is "pRU"?

> Also I always wondered how should we review these "reverse merge" PRs with many commits and changes?

https://github.com/Esri/arcgis-runtime-samples-ios/pull/869 very clearly shows which files are in conflict. So when reviewing a PR that fixes those conflicts, assuming updating from `v.next` was the only action taken, then only changes to those files would need to be reviewed (since all changes in other files are already reviewed prior to going into `v.next`). Does that make sense?

**Comment by yo1995** [comment]
Yes this makes sense. I'll also try with rebase to see if it is cleaner. Thanks for explanation! 😁

**Comment by yo1995** [comment]
Will close this and re-open #931

**Pull request closed without merging**